tests/src/JIT/Regression/JitBlue/GitHub_5047/project.json

```diff
@@ -0,0 +1,45 @@
+{
```

Comment:
Shouldn't there be a common app.config as well? I see that many tests have an app.config file but it's pretty much the same.